ComplexNumber.c

```diff
@@ -3,6 +3,31 @@
 ComplexNumber add(ComplexNumber first, ComplexNumber second) {
 	ComplexNumber result;
 	result.re = first.re + second.re;
-	result.re = first.im + second.im;
+	result.im = first.im + second.im;
+	return result;
+}
+
+ComplexNumber div(ComplexNumber first, ComplexNumber second) {
+	ComplexNumber result;
+
+	// Putting numbers as new variables for better reading
+	// This way first can be represented as (a + bi)
+	// And second -- (c + di)
+	double a = first.re;
+	double b = first.im;
+	double c = second.re;
+	double d = second.im;
+
+	result.re = (a*c + b*d) / (c*c + d*d);
+	result.im = (b*c - a*d) / (c*c + d*d);
```

Comment:
In theory, you can use abs-function here instead (c*c + d*d)
abs not implemented, but declared

Reply:
If I understand it correctly, we need to pow those numbers anyways, so using abs instead of `(c*c + d*d)` may give us incorrect values in the end